spec/system/user_tips_spec.rb

```diff
@@ -40,9 +40,9 @@
         text: "Your first notification!",
       )
 
-      find(".d-header").click
+      # Find an element with no action to simulate clicking outside the user tip
+      find("th.topic-list-data span", text: "Topic").click
```

Comment:
Picks a more specific element. In retrospect, `.d-header` was not a good choice, because it includes too many sub-elements and they can have their own actions and events.